samza-core/src/test/scala/org/apache/samza/container/TestTaskInstance.scala

```diff
@@ -48,7 +48,7 @@ import org.apache.samza.system.SystemAdmin
 import scala.collection.mutable.ListBuffer
 
 class TestTaskInstance {
-  @Test
+ /* @Test
```

Comment:
Same here. Please add comments and would prefer to move the changes to disable and fix the tests in a separate JIRA and PR.